index.html

```diff
@@ -159,119 +172,126 @@ <h3>Multi-Post Stories <br> Gain+Glory</h3>
             </div>
-            <div class="about_boxes">
-                <div class="box">
-                    <div> <img src="logo/icon-languages.png" alt="Languages"></div>
-                    <h2>Languages</h2>
-                    <ul>
-                        <li>JavaScript</li>
-                        <li>Ruby</li>
-                        <li>HTML</li>
-                        <li>CSS</li>
-                    </ul>
-                </div>
-                <div class="box">
-                    <div> <img src="logo/icon-frameworks.png" alt="Frameworks"></div>
-                    <h2>Frameworks</h2>
-                    <ul>
-                        <li>React.js</li>
-                        <li>Ruby on Rails</li>
-                        <li>RSpec</li>
-                    </ul>
-                    <ul>
-                        <li>Capybara</li>
-                        <li>Selenium</li>
-                    </ul>
-                </div>
-                <div class="box">
-                    <div> <img src="logo/icon-skills.png" alt="Skils"></div>
-                    <h2>Skills</h2>
-                    <ul>
-                        <li>Database Management</li>
-                        <li>Version Control</li>
 
-                    </ul>
-                    <ul>
-                        <li>CLI</li>
-                        <li>Web Development</li> 
-                        <li>API Design</li>
-                  
-                    </ul>
-                </div>
-            </div>
         </div>
+        <div class="about_boxes">
+            <div class="box">
+                <div> <img src="logo/icon-languages.png" alt="Languages"></div>
+                <h3>Languages</h3>
+                <ul>
+                    <li>JavaScript</li>
+                    <li>Ruby</li>
+                    <li>HTML</li>
+                    <li>CSS</li>
+                </ul>
+            </div>
+            <div class="box">
+                <div> <img src="logo/icon-frameworks.png" alt="Frameworks"></div>
+                <h3>Frameworks</h3>
+                <ul>
+                    <li>React.js</li>
+                    <li>Ruby on Rails</li>
+                    <li>RSpec</li>
+                </ul>
+                <ul>
+                    <li>Capybara</li>
+                    <li>Selenium</li>
+                </ul>
+            </div>
+            <div class="box">
+                <div> <img src="logo/icon-skills.png" alt="Skils"></div>
+                <h3>Skils</h3>
+                <ul>
+                    <li>Database Management</li>
+                    <li>Version Control</li>
 
+                </ul>
+                <ul>
+                    <li>CLI</li>
+                    <li>Web Development</li>
+                    <li>API Design</li>
+                </ul>
+            </div>
+        </div>
     </section>
-    <!-- Form-->
-    <section class="contact">
+
+    <section id="contact" class="contact">
```

Comment:
Well done on your contact me section! 👏👍 However, there are some updates needed for both the mobile and desktop versions. 

- [ ] In the desktop version, the line height of the section title does not match the design. Please adjust the line height of the section title to align with the design specifications. Additionally, the input fields are not aligned properly in comparison to the design. Please make the necessary adjustments to ensure the input fields are aligned as per the design.
- [ ] Moreover, the background image in the contact me section does not match the design. Please try to match the background image as closely as possible to maintain consistency with the design.
![Frame 3 (4)](https://github.com/Cheptoo21/Portfolio-setup-and-mobile-first/assets/55840999/7de503f4-07f8-422b-8f4c-72d7d83e4458)
- [ ] In the mobile version, similar to the desktop version, the line height of the section title needs to be updated. Additionally, the input fields require alignment adjustments to match the design.
![Frame 5 (1)](https://github.com/Cheptoo21/Portfolio-setup-and-mobile-first/assets/55840999/194194e4-1af5-4dec-a2fa-7f9427e18b6a)
By addressing these issues, you'll create a contact me section that is consistent with the design and enhances the overall user experience. Keep up the good work! 💪✨